app/src/main/kotlin/com/softteco/template/data/di/DataStoreModule.kt

```diff
@@ -57,6 +60,19 @@ object DataStoreModule {
         )
     }
 
+    @Provides
+    @Singleton
+    fun provideUserProfileEncryptedDataStore(@ApplicationContext appContext: Context): DataStore<ProfileDto> {
```

Comment:
If we create a new data store for keeping user data, then probably we can remove the old one (for CreateUserDto)?